nautobot/extras/tests/test_views.py

```diff
@@ -1679,8 +1679,8 @@ class RelationshipTestCase(
     ViewTestCases.DeleteObjectViewTestCase,
     ViewTestCases.EditObjectViewTestCase,
     ViewTestCases.BulkDeleteObjectsViewTestCase,
-    # TODO? ViewTestCases.GetObjectViewTestCase,
-    # TODO? ViewTestCases.GetObjectChangelogViewTestCase,
+    ViewTestCases.GetObjectViewTestCase,
+    ViewTestCases.GetObjectChangelogViewTestCase,
```

Comment:
Excellent!

Reply:
Thanks! I just figured I would test it with both test cases in.